src/site/markdown/server/manage-webadmin.md

```diff
@@ -591,6 +592,329 @@ Response codes:
  - 400: Group structure or member is not valid
  - 500: Internal error
 
+## Administrating mail repositories
```

Comment:
s/Administrating/Operating ?

Reply:
I don't really see the difference between both terms. Especially "Administrating" is used throughout the **WebAdmin** documentation page.

Any rational for such changes?